### ISSUE TITLE
Lua error for field "xp"

### ISSUE THREAD

**Issue by BreakBB** [Kreolix/CovenantMissionHelper#19]
With the latest version of CMH I get the following error:

```
Message: Interface\AddOns\CovenantMissionHelper\Board.lua:99: attempt to perform arithmetic on field 'xp' (a nil value)
Time: Sun Jan 24 22:35:57 2021
Count: 1
Stack: Interface\AddOns\CovenantMissionHelper\Board.lua:99: attempt to perform arithmetic on field 'xp' (a nil value)
[string "=[C]"]: ?
[string "@Interface\AddOns\CovenantMissionHelper\Board.lua"]:99: in function `new'
[string "@Interface\AddOns\CovenantMissionHelper\CovenantMissionHelper.lua"]:29: in function <...dOns\CovenantMissionHelper\CovenantMissionHelper.lua:25>
[string "@Interface\AddOns\CovenantMissionHelper\CovenantMissionHelper.lua"]:37: in function `simulateFight'
[string "@Interface\AddOns\CovenantMissionHelper\CovenantMissionHelper.lua"]:20: in function <...dOns\CovenantMissionHelper\CovenantMissionHelper.lua:15>
[string "=[C]"]: in function `UpdateAllyPower'
[string "@Interface\AddOns\Blizzard_GarrisonUI\Blizzard_CovenantMissionUI.lua"]:647: in function `AssignFollowerToMission'
[string "@Interface\AddOns\Blizzard_GarrisonUI\Blizzard_CovenantMissionUI.lua"]:472: in function `ptr'
[string "@Interface\SharedXML\CallbackRegistry.lua"]:55: in function `TriggerEvent'
[string "@Interface\AddOns\Blizzard_GarrisonUI\Blizzard_AdventuresPuck.lua"]:426: in function <...dOns\Blizzard_GarrisonUI\Blizzard_AdventuresPuck.lua:423>
```

This error appears when moving any "Troops"-Companion on the field.

Version: `CovenantMissionHelper-62fe4ea-alpha`

**Comment by Kreolix** [comment]
Fixed in v3.0.1 #18

**Comment by BreakBB** [comment]
Awesome, thanks 🎉
